src/components/EmptyState/WEmptyState.vue

```diff
@@ -0,0 +1,16 @@
+<template>
+  <div class="grid gap-md justify-center align-middle">
+    <EmptyStateLogo class="w-max" />
+    <p>{{ text }}</p>
+  </div>
+</template>
+
+<script setup>
```

Comment:
I think we could have problems importing this components ts language isn't specified
```suggestion
<script setup lang="ts">
```